File: sdks/go/pkg/models/operations/gettasksid.go
package operations

import (
	"openapi/pkg/models/shared"
	"openapi/pkg/utils"
)

type GetTasksIDPathParams struct {
	TaskID string `pathParam:"style=simple,explode=false,name=task_id"`
}

type GetTasksIDRequest struct {
	Retries    *utils.RetryConfig
	PathParams GetTasksIDPathParams
}

type GetTasksIDResponse struct {
	ClientError *shared.ClientError
	ContentType string
	StatusCode  int64
	Task        *shared.Task
}

File: sdks/ts/src/sdk/models/operations/gettasksid.ts
import { SpeakeasyMetadata, SpeakeasyBase, RetryConfig } from "../../../internal/utils";
import * as shared from "../shared";



export class GetTasksIdPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=task_id" })
  taskId: string;
}


export class GetTasksIdRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  retries?: RetryConfig;

  @SpeakeasyMetadata()
  pathParams: GetTasksIdPathParams;
}


export class GetTasksIdResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  clientError?: shared.ClientError;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  task?: shared.Task;
}
